scripts/tasks/screener.ts

```diff
@@ -37,6 +35,8 @@ export async function screener() {
     if (!affectedPackages.has(affectedPackageInfo.packageJson.name)) {
       await cancelScreenerRun(screenerConfig, 'skipped');
     } else {
+      const screenerStates = await getScreenerStates(screenerConfig);
```

Comment:
bad rebase? why is this needed for table ?

Reply:
whoops, no idea how it got there